tests/Type/ValidationTest.php

```diff
@@ -3103,6 +3103,26 @@ public function testRejectsASchemaWithDirectiveDefinedMultipleTimes()
         );
     }
 
+    public function testRejectsASchemaWithDirectivesWithWrongArgs()
```

Comment:
This case was bugged before, the combination of strict typing and code coverage led me to find this.

Reply:
Coverage really was essential here, found one more bug.